src/wallet/spend.cpp

```diff
@@ -211,13 +211,19 @@ void AvailableCoins(const CWallet& wallet, std::vector<COutput>& vCoins, const C
     }
 }
 
+// Allows the 'feerate' optional parameter of AvailableCoins to be skipped when utxos are not being spent
```

Comment:
This comment would fit better above the `AvailableCoinsListUnspent` declaration in wallet/spend.h so someone looking to understand how to use the interface at a glance can see it. Also, consider making it [doxygen-compatible](https://github.com/bitcoin/bitcoin/blob/master/doc/developer-notes.md#coding-style-doxygen-compatible-comments).

Reply:
done, I also clarified the comment a bit as well